### PR TITLE
Merge Milestone3 From Dev

### PR DESCRIPTION
**Pull request by MephistoDevelop**
Added features:

- Add Props from styles
- Add Flexbox
- Add PropTypes

### CODE REVIEW

File: src/assets/index.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  height: 100px;
+  color:white;
+  font-weight: bolder;
+  display: flex;
+justify-content: flex-end;
+align-items: center;
+font-size: 28px;
+padding: 0 10px;
+}
+#text-result{}
```
Comment (azdrenymeri) [inline src/assets/index.css:47]: This is an empty selector you can remove it

File: src/components/Buttons.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/default-props-match-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ButtonPanel = ({ buttonName, Color, Width }) => {
```
Comment (azdrenymeri) [inline src/components/Buttons.js]: `buttonName`,`color`, and `width` props should be moved to `Button` component.

File: src/components/Buttons.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  const ButtonName = buttonName;
+  let styles = { backgroundColor: Color, width: Width };
+  if (Width) {
+    styles = { backgroundColor: Color, width: '209%' };
+  }
```
Comment (azdrenymeri) [inline src/components/Buttons.js]: if `Width` is true the `width` of the button should be `50%` not `209%`.

**Review by azdrenymeri — CHANGES_REQUESTED** [review]
### Hello @MephistoDevelop :wave: 
#### There are some changes that need to be done on `milestone2` so we can be able to do a proper review of this milestone.

- [x] There should be a `Button` component.
- [x] `Button` component  take 25% of container width (except `0` button)
scr
- [x] `Button` component accepts `color` and `wide` as an incoming props
- [x] `wide` is boolean, true means that button take 50% of space (used for `0` button)
- [x] `Button` coponent's defualt `color` is orange

### Kindly implement the changes and add it for another review.
### If anything hit me up on slack :+1:

File: src/components/Buttons.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/default-props-match-prop-types */
+import React from 'react';
+import Button from './button';
+
```
Comment (shloch) [inline src/components/Buttons.js]: hello bro, please rename then this file as `BottonPanel.js`, to not confuse with the other file. Hope you understand this

**Review by shloch — CHANGES_REQUESTED** [review]
## hello @MephistoDevelop 

great job

### [required changes]

Please few more things and we're set to go.
-  please rename file mentioned below for organisation..... 😊😊

- your `.eslintrc.json` file should be same as that of microverse' [ [click here to see](https://github.com/shloch/linters-config/blob/master/react-redux/.eslintrc.json) ]


I know i have added few additional tasks, it's just to stick with the linting assigned for the program. There'll be no more to it......and we're finishing ...thanks for understand 😊😊

Happy coding.......

File: src/components/Buttons.js (hunk 4 above)
Comment (MephistoDevelop) [reply src/components/Buttons.js]: ok, i understand that points and i already did hat changes. Thanks for the review

**Review by shloch — APPROVED** [review]
accomplished

**Comment by MephistoDevelop** [comment]
jajaj thanks !! i love hat pic lol

**Pull request merged**